### PR TITLE
Control Plane Lite documentation update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added relevant notes/warnings re: maintenance and setup
> Tweaked/polished some content for better readability.

Update: per convo with @saramashfej : Main point is to strongly admonish against using control plane lite but also note that this is an open source project and that contributions are welcome

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/get-started/rudderstack-open-source/control-plane-lite.mdx [hunk 1]
```diff
@@ -258,19 +247,17 @@ The SDKs will then fetch the config from `<CONTROL_PLANE_URL>/sourceConfig`. A s
 
 ## FAQ
 
-### For a self-hosted environment, how do I obtain the control plane URL?
+### How do I obtain the control plane URL for a self-hosted environment?
+
+<div class="infoBlock">
+This solution assumes that you have already set up the RudderStack data plane (backend) locally.
```
Comment (shekhar-rudder) [inline docs/get-started/rudderstack-open-source/control-plane-lite.mdx:253]: Is this relevant to the question? Data plane setup is not a precondition for obtaining the CP url

Comment (ameypv-rudder) [reply docs/get-started/rudderstack-open-source/control-plane-lite.mdx:253]: Makes sense. I will remove this note.

File: docs/get-started/rudderstack-open-source/control-plane-lite.mdx [hunk 2]
```diff
@@ -258,19 +247,17 @@ The SDKs will then fetch the config from `<CONTROL_PLANE_URL>/sourceConfig`. A s
 
 ## FAQ
 
-### For a self-hosted environment, how do I obtain the control plane URL?
+### How do I obtain the control plane URL for a self-hosted environment?
+
+<div class="infoBlock">
```
Comment (ameypv-rudder) [inline docs/get-started/rudderstack-open-source/control-plane-lite.mdx:252]:
```suggestion
```

File: docs/get-started/rudderstack-open-source/control-plane-lite.mdx (hunk 1 above)
Comment (ameypv-rudder) [inline docs/get-started/rudderstack-open-source/control-plane-lite.mdx:253]:
```suggestion
```

File: docs/get-started/rudderstack-open-source/control-plane-lite.mdx [hunk 3]
```diff
@@ -258,19 +247,17 @@ The SDKs will then fetch the config from `<CONTROL_PLANE_URL>/sourceConfig`. A s
...
+This solution assumes that you have already set up the RudderStack data plane (backend) locally.
+</div>
```
Comment (ameypv-rudder) [inline docs/get-started/rudderstack-open-source/control-plane-lite.mdx:254]:
```suggestion
```

**Review by shekhar-rudder — COMMENTED** [review]
LGTM

File: docs/get-started/rudderstack-open-source/control-plane-lite.mdx [hunk 4]
```diff
@@ -1,52 +1,50 @@
...
-<div class="warningBlock">
-The control plane set up using Control Plane Lite does not support Live Events and Transformations.
+<div class="dangerBlock">
+<a href="https://github.com/rudderlabs/config-generator">Control Plane Lite</a> is no longer actively maintained. As a result, some of the <a href="https://app.rudderstack.com/signup">RudderStack Cloud</a> features may not be available in your self-hosted control plane.
 </div>
 
 ## Setting up the control plane
 
-To set up your self-hosted control plane using the Control Plane Lite utility, follow these steps:
+Before setting up your self-hosted control plane, make sure you have installed <a href="https://nodejs.org/en/download/">Node.js</a>.
```
Comment (ishachopra0401) [inline docs/get-started/rudderstack-open-source/control-plane-lite.mdx:21]:
```suggestion
Before setting up your self-hosted control plane, make sure you have installed <a href="https://nodejs.org/en/blog/release/v14.17.3/">Node.js</a>.
```
As downloading this version might not work

File: docs/get-started/rudderstack-open-source/control-plane-lite.mdx [hunk 5]
```diff
@@ -1,52 +1,50 @@
...
 
-<div class="infoBlock">
-
-Make sure you have <a href="https://nodejs.org/en/download/">Node.js</a> installed before setting up your self-hosted control plane.
+<div class="warningBlock">
+Node.js v16 and higher are not supported. For the best experience, using Node.js v14 is strongly recommended.
```
Comment (ishachopra0401) [inline docs/get-started/rudderstack-open-source/control-plane-lite.mdx:24]:
```suggestion
Use Node.js v14 for the best experience as the higher versions are not supported. 
```

File: docs/get-started/rudderstack-open-source/control-plane-lite.mdx [hunk 6]
```diff
@@ -258,19 +247,14 @@ The SDKs will then fetch the config from `<CONTROL_PLANE_URL>/sourceConfig`. A s
 
 ## FAQ
 
-### For a self-hosted environment, how do I obtain the control plane URL?
+### How do I obtain the control plane URL for a self-hosted environment?
```
Comment (ishachopra0401) [inline docs/get-started/rudderstack-open-source/control-plane-lite.mdx:250]: can skip as same steps are present in the above section

File: docs/get-started/rudderstack-open-source/control-plane-lite.mdx [hunk 7]
```diff
@@ -158,41 +156,32 @@ cp config/sample.env .env
...
-<div class="infoBlock">
-
-To know more about the difference between cloud mode and device mode in RudderStack, read the <Link to="/destinations/rudderstack-connection-modes/">RudderStack Connection Modes</Link> guide.
-</div>
-
-<div class="infoBlock">
-For RudderStack Cloud, the SDKs fetch this source configuration from <code class="inline code">https://api.rudderlabs.com/sourceConfig</code>.
-</div>
-
 ### Control plane URL
```
Comment (ishachopra0401) [inline docs/get-started/rudderstack-open-source/control-plane-lite.mdx:159]:
```suggestion
## Control plane URL
```

**Review by pfd — APPROVED** [review]
Made some minor edits to the callouts 👍🏻
